src/Stratis.Bitcoin.IntegrationTests/EnvironmentMockUpHelpers/NodeBuilder.cs

```diff
@@ -105,9 +104,10 @@ public class NodeBuilder : IDisposable
 
         public NodeConfigParameters ConfigParameters { get; }
 
+        private List<IDisposable> disposables;
         private int last;
         private string root;
-        private List<IDisposable> disposables;
+        private bool skipRules = false;
```

Comment:
why do you need this property?

Reply:
It is used to determine whether or not the node should skip consensus rules. Do a find all references, you;ll see that we can now set up the node in the integration tests like that.

`var node = nodeBuilder.SkipRules().CreateStratisPosNode()'

We then inject a fake rule registration class.